### PR TITLE
New frontend structure

### PR DESCRIPTION
**Pull request by ayyazdaniaryan**
Suggested new structure for developing the new front-end

### CODE REVIEW

File: assembl/__init__.py [hunk 1]
```diff
@@ -91,8 +92,9 @@ def main(global_config, **settings):
         from .lib.migration import bootstrap_db_data
         bootstrap_db_data(session, False)
 
+    config.add_static_view('react', 'react', cache_max_age=3600)
```
Comment (swergas) [inline assembl/__init__.py]: I'm really not sure the word "react" should appear in the URL...

File: assembl/lib/frontend_urls.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def __init__(self, discussion):
 
     @classmethod
     def register_frontend_routes(cls, config):
-        from assembl.views.backbone.views import home_view
+        from assembl.views.discussion.views import home_view
```
Comment (swergas) [inline assembl/lib/frontend_urls.py]: Why make the word "react" appear clearly in the architecture and not the word "backbone"? We should probably choose none or both?

File: assembl/views/__init__.py [hunk 3]
```diff
@@ -38,7 +38,8 @@
 
 
 default_context = {
-    'STATIC_URL': '/static'
+    'STATIC_URL': '/static',
+    'REACT_URL': '/react'
```
Comment (swergas) [inline assembl/views/__init__.py]: Same remark: I'm really not sure the word "react" should appear in the URL...

File: assembl/__init__.py (hunk 1 above)
Comment (paolina92) [inline assembl/__init__.py]: Suggestions to replace "react":
- static_v2
- frontend
- assembl (but the occurence will be present 3 times)

File: assembl/lib/frontend_urls.py (hunk 2 above)
Comment (paolina92) [inline assembl/lib/frontend_urls.py]: In my opinion "discussion" is ok

File: assembl/views/__init__.py (hunk 3 above)
Comment (paolina92) [inline assembl/views/__init__.py]: see my suggestions in the my other comment

File: assembl/lib/frontend_urls.py (hunk 2 above)
Comment (maparent) [inline assembl/lib/frontend_urls.py]: I think so too. this module contains both backbone and react views, hence the rename. (plus, backbone vs home was confusing)

File: assembl/__init__.py (hunk 1 above)
Comment (ayyazdaniaryan) [inline assembl/__init__.py]: @swergas @paolina92 `static2` sound okay?

Comment (ayyazdaniaryan) [inline assembl/__init__.py]: It was just a suggestion to use `react`. Open for a change.

Comment (paolina92) [inline assembl/__init__.py]: yes static2 okay!

**Comment by ayyazdaniaryan** [comment]
I changed the folder name to static2, the main route to access react is still `server:port/v2/discussion_slug`.

**Pull request merged**